harper-core/tests/run_tests.rs

```diff
@@ -36,13 +39,18 @@ macro_rules! create_test {
     };
 }
 
-create_test!(whack_bullets.md, 1);
-create_test!(preexisting.md, 0);
-create_test!(issue_109.md, 0);
-create_test!(issue_109_ext.md, 0);
-create_test!(chinese_lorem_ipsum.md, 2);
-create_test!(obsidian_links.md, 2);
-create_test!(issue_267.md, 0);
-create_test!(proper_noun_capitalization.md, 2);
-create_test!(amazon_hostname.md, 0);
-create_test!(issue_159.md, 1);
+create_test!(whack_bullets.md, &Markdown, 1);
+create_test!(preexisting.md, &Markdown, 0);
+create_test!(issue_109.md, &Markdown, 0);
+create_test!(issue_109_ext.md, &Markdown, 0);
+create_test!(chinese_lorem_ipsum.md, &Markdown, 2);
+create_test!(obsidian_links.md, &Markdown, 2);
+create_test!(issue_267.md, &Markdown, 0);
+create_test!(proper_noun_capitalization.md, &Markdown, 2);
+create_test!(amazon_hostname.md, &Markdown, 0);
+create_test!(issue_159.md, &Markdown, 1);
+
+#[cfg(feature = "typst")]
+create_test!(complex_typst.typ, &Typst, 0);
+#[cfg(feature = "typst")]
+create_test!(typst_spelling_mistakes.typ, &Typst, 4);
```

Comment:
Why is this line incomplete?

Reply:
Made a mistake just now when merging through github interface, just pushed a fix.